test/basic.test.js

```diff
@@ -49,7 +49,7 @@ test('basic prettifier tests', (t) => {
         const formatted = pretty(chunk.toString())
         t.is(
           formatted,
-          `[${epoch}] INFO (${pid} on ${hostname}): foo\n`
+          `[${epoch}] INFO  (${pid} on ${hostname}): foo\n`
```

Comment:
Can you revert every change in this file except for the relevant ones to this PR?

Reply:
They're reverted, have a look at the changes:
https://github.com/pinojs/pino-pretty/pull/134/files